src/AutomatedCar/Models/IWorldObject.cs

```diff
@@ -1,31 +0,0 @@
-﻿namespace AutomatedCar.Models
-{
-    using System.Collections.ObjectModel;
-    using System.Drawing;
-    using Avalonia.Media;
-
-    public interface IWorldObject
```

Comment:
Miért lettek ezek az interface-k törölve egészen pontosan?

Reply:
Mi hoztuk létre anno az összeset amit most töröltünk

Eredetileg mockoláshoz voltak használva, viszont azok a tesztek amik hasznalták őket már nincsenek a projektben (refaktorálás miatt értelmüket vesztették)

Mivel senki más nem használta őket, és már mi sem ezért kivettük őket.

Eleve nem is voltak rendesen karbantarva.(Iworldobject interfész pl nem tartalmazta az összes publikus worldobject adattagot)

Reply:
Az enumerable-k nekünk is gondot okoztak, szóval köszönjük, hogy cseréltétek őket.

Az interfészeket én a legújabb fejlesztésben elkezdtem használni, de nem komoly a cseréjük, elsősorban ezért érdeklődtem.

Megy az approve.